MAUI/DataForm/floating-label-layout.md

```diff
@@ -868,4 +870,6 @@ N> [View sample in GitHub](https://github.com/SyncfusionExamples/maui-dataform/t
 ## Unsupported editors
 The Floating label layout does not support non-editable editors such as [RadioGroup editor](https://help.syncfusion.com/maui/dataform/editors#radiogroup-editor), [CheckBox editor](https://help.syncfusion.com/maui/dataform/editors#checkbox-editor), and [Switch editor](https://help.syncfusion.com/maui/dataform/editors#switch-editor).
 
-N> By default, the layout type of unsupported editors is `Default`. You can also set the layout type for the unsupported editor using this [Changing layout type of the DataFormItem](https://help.syncfusion.com/maui/dataform/floating-label-layout#changing-the-layout-type-for-the-editor).
\ No newline at end of file
+N> By default, the layout type of unsupported editors is `Default`. You can also set the layout type for the unsupported editor using this [Changing layout type of the DataFormItem](https://help.syncfusion.com/maui/dataform/floating-label-layout#changing-the-layout-type-for-the-editor).
+
+N> The time picker will not work in windows platform within text input layout.
```

Comment:
N> The time picker will not work in the Windows platform within the text input layout.

Reply:
Updated